### PR TITLE
Update to Blazor 0.2.0

### PR DESCRIPTION
**Pull request by panesofglass**
Blazor 0.2.0 was [just announced](https://blogs.msdn.microsoft.com/webdev/2018/04/17/blazor-0-2-0-release-now-available/)! This PR updates everything to use 0.2.0.

Outstanding issues:
- [x] Travis build failing
- [x] [Blazor-Redux](https://github.com/torhovland/blazor-redux) [sample](https://github.com/panesofglass/trail/tree/upgrade/blazor-0.2.0/sample/blazor-redux-standalone/) fails to build. @torhovland, have you had any issues with 0.2.0? Will I need to await an upgraded package?

### CODE REVIEW

File: src/Trail/Trail.fs [hunk 1]
```diff
@@ -53,6 +56,21 @@ module Dom =
     
     let text textContent = Text textContent
     let textf format content = Text(Printf.sprintf format content)
+
+module Attr =
+    open Dom
+    
+    let namedUIEvent name (handler:string) =
+        BlazorEventAttribute(name, BindMethods.GetEventHandlerValue(handler))
```
Comment (panesofglass) [inline src/Trail/Trail.fs:64]: Not sure this one should remain.

File: src/Trail/Trail.fs [hunk 2]
```diff
@@ -53,6 +56,21 @@ module Dom =
...
+    
+    let uiEventHandler name handler =
+        BlazorEventHandlerAttribute(name, BindMethods.GetEventHandlerValue(Action handler))
+
+    let typedUIEventHandler<'T when 'T :> UIEventArgs> name handler =
+        BlazorEventHandlerAttribute(name, BindMethods.GetEventHandlerValue(Action<'T> handler))
+    
+    let onclick handler =
+        typedUIEventHandler<UIMouseEventArgs> "onclick" handler
```
Comment (panesofglass) [inline src/Trail/Trail.fs:73]: There are more examples in the [blog post](https://blogs.msdn.microsoft.com/webdev/2018/04/17/blazor-0-2-0-release-now-available/) that I should probably add.

File: src/Trail/Trail.fs [hunk 3]
```diff
@@ -67,6 +85,8 @@ module RenderTree =
         | AddBlazorFragmentAttribute of sequence:int * fragment:AST list
         | AddBlazorFrameAttribute of sequence:int * frame:RenderTree.RenderTreeFrame
         | AddBlazorObjAttribute of sequence:int * name:string * value:obj
+        | AddBlazorEventAttribute of sequence:int * name:string * handler:string
```
Comment (panesofglass) [inline src/Trail/Trail.fs:88]: May not need this option.

File: src/Trail/Trail.fs [hunk 4]
```diff
@@ -6,8 +6,11 @@ open Microsoft.AspNetCore.Blazor.Components
 open Microsoft.AspNetCore.Blazor.Routing
 
 module Dom =
+
     type Attribute =
         | HtmlAttribute of name:string * value:string
+        | BlazorEventAttribute of name:string * handler:string
```
Comment (panesofglass) [inline src/Trail/Trail.fs:12]: May not need this option.

**Comment by torhovland** [comment]
@panesofglass Blazor-Redux has been updated for Blazor 0.2.0 now. Hope that helps.

**Pull request merged**